src/Fixer/Phpdoc/PhpdocTagNoNamedArgumentsFixer.php

```diff
@@ -0,0 +1,233 @@
+use PhpCsFixer\FixerDefinition\FileSpecificCodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Preg;
+use PhpCsFixer\Tokenizer\Analyzer\Analysis\NamespaceUseAnalysis;
+use PhpCsFixer\Tokenizer\Analyzer\AttributeAnalyzer;
+use PhpCsFixer\Tokenizer\Analyzer\FullyQualifiedNameAnalyzer;
+use PhpCsFixer\Tokenizer\CT;
+use PhpCsFixer\Tokenizer\FCT;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+use PhpCsFixer\WhitespacesFixerConfig;
+
+/**
+ * @phpstan-type _InputConfig array{directory?: string, description?: string}
+ * @phpstan-type _Config array{directory: string, description: string}
+ *
+ * @no-named-arguments
+ *
+ * @phpstan-type _AutogeneratedInputConfiguration array{
+ *  description?: string,
+ *  directory?: string,
+ * }
+ * @phpstan-type _AutogeneratedComputedConfiguration array{
+ *  description: string,
+ *  directory: string,
+ * }
+ *
+ * @implements ConfigurableFixerInterface<_AutogeneratedInputConfiguration, _AutogeneratedComputedConfiguration>
+ */
+final class PhpdocTagNoNamedArgumentsFixer extends AbstractFixer implements ConfigurableFixerInterface, WhitespacesAwareFixerInterface
+{
+    private string $description = '';
+    private string $directory = '';
+
+    public function setWhitespacesConfig(WhitespacesFixerConfig $config): void
+    {
+        $this->whitespacesConfig = $config;
+    }
+
+    public function getDefinition(): FixerDefinitionInterface
+    {
+        return new FixerDefinition(
+            'There must be `@no-named-arguments` tag in PHPDoc of a class/enum/interface/trait.',
+            [
+                new FileSpecificCodeSample(
+                    <<<'PHP'
+                        <?php
+                        class Foo
+                        {
+                            public function bar(string $s) {}
+                        }
+
+                        PHP,
+                    new \SplFileInfo(__DIR__.'/../..'),
+                ),
+                new FileSpecificCodeSample(
+                    <<<'PHP'
+                        <?php
+                        class Foo
+                        {
+                            public function bar(string $s) {}
+                        }
+
+                        PHP,
+                    new \SplFileInfo(__DIR__.'/../..'),
+                    ['description' => 'the reason'],
+                ),
+            ],
+        );
+    }
+
+    public function getConfigurationDefinition(): FixerConfigurationResolverInterface
+    {
+        return new FixerConfigurationResolver([
+            (new FixerOptionBuilder('description', 'Description of the tag.'))
+                ->setAllowedTypes(['string'])
+                ->setDefault('')
+                ->getOption(),
+            (new FixerOptionBuilder('directory', 'Directory in which apply the changes, empty value will result with current working directory (result of `getcwd` call).'))
```

Comment:
what's the use case for this option ?

Reply:
I was inspired to create the fixer by 2 things:
 - https://php.watch/articles/no-named-arguments-docblock-attribute
 - I saw them in PHPUnit

When I created it, I tested it on PHPUnit codebase, and since @sebastianbergmann has the fixtures directory under coding standards check, it created a lot of changes, so I have added the option.

Sebastian, would you be fine with having `@no-named-arguments` (or `@internal`) tag in all the test fixtures, or would you use the `directory` option?